tests/data/planetary-computer-root.json

```diff
@@ -59,6 +59,7 @@
     ],
     "conformsTo": [
         "https://api.stacspec.org/v1.0.0-beta.1/core",
+        "https://api.stacspec.org/v1.0.0-beta.1/collections",
```

Comment:
As of this writing, the planetary computer tracks `rc.1`. My instinct is to wait until `v1` before updating the text fixtures, but wanted to check w/ you.

Reply:
the regex should work with any of them, so no need to update `re.escape("https://api.stacspec.org/v1.0.")`